sdk/test-utils/perfstress/runAllTests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+rushx clean
+rushx build
+npm run unit-test:node -- OptionsTest           --warmup 2 --duration 4 --iterations 2
+npm run unit-test:node -- Delay500ms            --warmup 2 --duration 4 --iterations 2
+npm run unit-test:node -- SynchronousException  --warmup 2 --duration 4 --iterations 2
+npm run unit-test:node -- AsynchronousException --warmup 2 --duration 4 --iterations 2
+npm run unit-test:node -- PerfStressPolicyTest  --warmup 2 --duration 4 --iterations 2
```

Comment:
I'm using this to make sure that all of the tests pass. It's a rough utility. I'm thinking on cleaning this later.